test/es6/default.js

```diff
@@ -203,7 +203,7 @@ var tests = [
     }
   },
   {
-    name: "OS 1583694: Arguments sym is not set correctly on undo defer",
+    name: "OS 1583694: Arguments sym is not set correctly on undo defer",
     body: function () {

```

Comment:
Looks like a whitespace only change here.  I can't see what it is, did the type of newline change?